hack/prebuild/cluster_setup.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
Could this file be moved to `hack/aws-provsion.sh` and also run terraform? This will create a simple path to include it into Makefile